extensions/2.0/Vendor/EXT_materials_bump/README.md

```diff
@@ -0,0 +1,142 @@
+# EXT\_materials\_bump
+
+## Contributors
+
+- Ben Houston, Threekit [@bhouston](https://github.com/bhouston)
+
+Copyright 2023 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft, Ratified by the Khronos Group
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Exclusions
+
+- This extension must not be used on a material that also uses `KHR_materials_pbrSpecularGlossiness`.
+- This extension must not be used on a material that also uses `KHR_materials_unlit`.
+
+## Overview
+
+This extension adds one parameters to the metallic-roughness material: `bump`.
+
+`bump` allows users to perturb the surface normal via a bump map. The bump map is a grayscale texture, where the value of each texel represents the height of the surface at that texel.  The height is then used to perturb the normal direction.
```

Comment:
I think "grayscale" will need clarification here. If this is a single-channel texture, can we specify which channel the implementation must sample from, and allow other channels to be used by other material texture slots? Probably that should be the red channel - see below.

Reply:
We should specify the channel.  I was thinking it may be possible to pack in into another channel (like the ORM packing) so I sort of held off on that.  But if we want it to be its own map, red makes sense.

Reply:
I agree, I have made the default channel A as you suggest.

Reply:
> I agree, I have made the default channel A as you suggest.

Did you mean `R`?